tasks/task1.ipynb

```diff
@@ -46,29 +49,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": null,
    "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<festim.meshing.mesh_from_vertices.MeshFromVertices at 0x7f91d7a5d1c0>"
-      ]
-     },
-     "execution_count": 33,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
+   "outputs": [],
    "source": [
     "F.MeshFromVertices(vertices=[0, 1, 2, 3, 4, 5, 6, 7, 7.5])"
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Numpy can be used to generate heavier meshes. Here we create a mesh containing 1000 cells over a [0, 1e-6] domain (1 micron).\n",
+    "Numpy can be used to generate heavier meshes. Here we create a mesh containing 1000 cells over a [0, 1e-6] domain (1 $\\mu m$).\n",
```

Comment:
\mu is not a symbol for micron!!!

Reply:
Turned out we can use \micro when using the package \siunitsx but will keep this open incase we find another way to have it in text form in markdown